CMakeLists.txt

```diff
@@ -205,7 +206,7 @@ cmake_dependent_option(OPENSCAP_PROBE_INDEPENDENT_XMLFILECONTENT "Independent xm
 # UNIX PROBES
 cmake_dependent_option(OPENSCAP_PROBE_UNIX_DNSCACHE "Unix dnscache probe" ON "ENABLE_PROBES_UNIX" OFF)
 cmake_dependent_option(OPENSCAP_PROBE_UNIX_FILE "Unix file probe" ON "ENABLE_PROBES_UNIX" OFF)
-cmake_dependent_option(OPENSCAP_PROBE_UNIX_FILEEXTENDEDATTRIBUTE "Unix fileextendedattribute probe" ON "ENABLE_PROBES_UNIX" OFF)
+cmake_dependent_option(OPENSCAP_PROBE_UNIX_FILEEXTENDEDATTRIBUTE "Unix fileextendedattribute probe" ON "ENABLE_PROBES_UNIX; (HAVE_SYS_XATTR_H OR HAVE_ATTR_XATTR_H)" OFF)
```

Comment:
Where are those 2 variables defined?

Reply:
@jan-cerny: https://github.com/OpenSCAP/openscap/pull/1118/files#diff-57fb65583e7c90f3400179b1a4319f42R78 (config.in)

Reply:
@cipherboy 

Not really, In fact that's an usage of these variables. CMake uses them to generate `config.h` from `config.h.in`. 

The problem is they need to be defined somewhere in CMakeLists.txt to make this `cmake_dependent_option` working. I'm afraid that now the fileextendedattribute probe does not build at all.

Most likely there should be on line 151:
```
check_include_file(sys/xattr.h HAVE_SYS_XATTR_H)
check_include_file(attr/xattr.h HAVE_ATTR_XATTR_H)
```

Reply:
hmm, I am really puzzled by this because I have it locally, I guess I just didn't commit that hunk